src/lib/utils/pageSize.ts

```diff
@@ -43,21 +43,27 @@ export function pageSizesFromSpec(pageSpec) {
  * Parse page_spec and return an array of [width, height] tuples
  *
  * @param {string} pageSpec A string encoding page dimensions in a compact format
```

Comment:
This file looks like an unrelated change

Reply:
It is. Just had some type annotations I'd meant to put in there, so I rolled it into this.